bob-access/src/authenticator/users_storage/mod.rs

```diff
@@ -30,15 +30,15 @@ impl Perms {
 
 #[derive(Debug, Clone, Deserialize, Default)]
 pub struct User {
-    username: String,
+    _username: String,
```

Comment:
Why the underscore symbol was added before name here and in other places?

Reply:
It is not used in this branch. Unused vars should be prefixed with `_`